.github/workflows/release-upload.yaml

```diff
@@ -94,29 +96,44 @@ jobs:
       timeout-minutes: 120
       run: |
         while true; do
-          # When supporting other architectures than Linux, this query should be adapted:
-          conclusion=$(gh api "repos/$GITHUB_REPOSITORY/commits/${{ env.TARGET_TAG }}/check-runs" --jq '.check_runs[] | select(.name | test("ci/hydra-build:.*-linux.required")) | .conclusion')
-          case "$conclusion" in
-            success)
-              echo "ci/hydra-build:required succeeded"
-              exit 0;;
-            failure)
-              echo "ci/hydra-build:required failed"
-              exit 1;;
-            *)
-              echo "ci/hydra-build:required pending. Waiting 30s..."
-              sleep 30;;
-          esac
+          conclusion=$(gh api "repos/$GITHUB_REPOSITORY/commits/${{ env.TARGET_TAG }}/check-runs" --jq '.check_runs[] | select(.name | test("ci/hydra-build:.*\\.required")) | .conclusion')
```

Comment:
Previously we were matching a single job, whereas now we match 3 of them; so we need a more flexible handling, hence the new lines below.